src/biconnectivity/articulation.jl

```diff
@@ -78,3 +78,59 @@ function articulation(g::AbstractGraph)
     end
     return findall(state.articulation_points)
 end
+
+function iterative_articulation(g::SimpleGraph)
+    T = eltype(g)
+    s = Vector{Tuple{Int, Int, T, T}}()
+    articulation_points = Set{T}()
+    low = zeros(Int, nv(g))
+    pre = zeros(Int, nv(g))
```

Comment:
You can replace these by `zeros(T, nv(g))`, right?

Reply:
Yes i forgot this is incorrect.But if we have to consider all graph types then shouldn't we use Dict{eltype(g),Int} ? but this will slow down the function.Should I just stick to Int Graphs ?

Reply:
`cnt` is assigned to each vertex. I think this should work for all graphs since the array index is integer and all SimpleGraph types <: Integer